metricbeat/mb/testing/data/data_test.go

```diff
@@ -219,7 +219,7 @@ func checkDocumented(t *testing.T, data []common.MapStr) {
 				if _, ok := keys[prefix+".*"]; ok {
 					continue
 				}
-				t.Fatalf("key missing: %s", k)
+				t.Fatalf("check if fields are documented error: key missing '%s'", k)
```

Comment:
@ruflin I added this because the original message did not show very clearly what was failing and it took me a while to figure out (until I checked the error line in fact). I hope this is ok :)